lib/inv_ca_gcr.cpp

```diff
@@ -248,7 +246,7 @@ namespace quda {
       // build up a space of size nKrylov
       for (int k=0; k<nKrylov; k++) {
         matSloppy(*q[k], *p[k], tmpSloppy);
-        if (k<nKrylov-1 && basis != POWER_BASIS) blas::copy(*p[k+1], *q[k]);
+        if (k<nKrylov-1 && basis != QUDA_POWER_BASIS) blas::copy(*p[k+1], *q[k]);
```

Comment:
clang-format suggested the following change:


```
-        if (k<nKrylov-1 && basis != QUDA_POWER_BASIS) blas::copy(*p[k+1], *q[k]);
+        if (k < nKrylov - 1 && basis != QUDA_POWER_BASIS) blas::copy(*p[k + 1], *q[k]);
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_gcr.cpp -->